packages/json-api/src/lib/query/stringifyQuery.ts

```
@@ -0,0 +1,194 @@
+          if (isFilterOperator(value)) {
+            const ops: NonNullable<ClientJsonApiQuery["filter"]>[string] = {};
+            for (const [op, v] of Object.entries(value)) {
+              if (v !== undefined) {
+                const filterValues = toFilterValue(v);
+                if (filterValues.length > 0) {
+                  ops[op as FilterOperator] = filterValues;
+                }
+              }
+            }
+
+            return Object.keys(ops).length > 0 ? { ...accumulator, [key]: ops } : accumulator;
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)